apps/frontend/src/pages/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -1,16 +1,111 @@
-import { Button } from "~/libs/components/components.js";
+import { Link } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+import { userSignInValidationSchema } from "shared";
+import { type UserSignInRequestDto } from "shared/src/modules/users/libs/types/types.js";
```

Comment:
same comment about direct import from shared folder. Check it in the rest of code in this PR